addons/stock/models/stock_quant.py

```diff
@@ -656,6 +656,7 @@ class QuantPackage(models.Model):
     _name = "stock.quant.package"
     _description = "Packages"
     _order = 'name'
+    _rec_name = 'name'
```

Comment:
Already done by default
https://github.com/odoo/odoo/blob/a57d4377fb2706e4d21a86cd01e9556b0e2501ba/odoo/models.py#L2808

Reply:
Yes, right, it will a better diff